cmd/registry/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"github.com/apigee/registry/rpc"
+)
+
+// ComputeMetrics will compute the metrics proto for a list of Classified Diffs.
+func ComputeMetrics(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Metrics {
+	stats := calculateStats(calculatedDiffs)
+	return caluclateMetrics(stats)
+}
+
+func calculateStats(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Stats {
+	totalBreakingChanges := 0
+	totalNonbreakingChanges := 0
+	for _, caluclatedDiff := range calculatedDiffs {
```

Comment:
Nit: `caluclatedDiff` -> `calculatedDiff`.

As an aside, I think these names are fairly verbose and can be clear with smaller names. For example, `calculatedDiffs` -> `diffs`, `totalBreakingChanges` -> `breaking`, `totalNonbreakingChanges` -> `nonbreaking`, `calculatedDiff` -> `diff`.

Reply:
Done